mods/cnc/chrome/editor.yaml

```diff
@@ -366,8 +367,8 @@ Container@EDITOR_WORLD_ROOT:
 					Height: PARENT_BOTTOM
 					Children:
 						Background:
-							Width: 61
-							Height: 75
+							Width: 60
+							Height: 73
```

Comment:
rebase error?

Reply:
Regarding that, I found the size of a panel in the editor is slightly incorrect, and thought I would fix it while I was here. But I think I might back it out instead. Thanks